### ISSUE TITLE
Export to AsciiDoc

### ISSUE THREAD

**Issue by jmfayard** [fabiospampinato/notable#163]
Hello,
It would be very useful for me to have an export from markdown to AsciiDoc. 
If you are not familiar, asciidoc is similar to markdown for the easy things but doesn't break down as soon as you want to do something more complex with it.  
https://asciidoctor.org/docs/user-manual/#compared-to-markdown

**Comment by mnikn** [comment]
Seems like pandoc can convert it.

**Comment by Orcomp** [comment]
It would be great if Notable supported AsciiDoc out of the box so you would not have to export anything ;) I agree AsciiDoc is much more flexible and powerful than Markdown. I loved and used Markdown for the last 7 years but found it lacking in a few areas. AsciiDoc was very easy to transition to and has all the features I need. If Notable was able to support both formats it would be a match made in heaven

**Comment by fabiospampinato** [comment]
@Orcomp I have no use for AsciiDoc so I don't think this is going to get implement, at least not in the foreseeable future.

**Comment by 5paceToast** [comment]
As a side-note, asciidoc has a reference implementation in javascript: [asciidoctor.js](https://github.com/asciidoctor/asciidoctor.js).
Supporting asciidoc (whether as an export target or a native format) is likely much less of a pain than one might anticipate, as a direct result.

I have the same view as @Orcomp, and think that asciidoc support + some sort of mobile editing would really bring notable far ahead of everything else available.

**Comment by smvoss** [comment]
Just looked at this and I saw help wanted was removed, has this had a state change?

**Comment by fabiospampinato** [comment]
@smvoss Not really, I removed the label just because newer versions of the app aren't really being released as open-source (I'll make this clearer), so PRs would have to made against an older version of the app which is not that useful.

**Comment by fabiospampinato** [comment]
If anyone knows of a good NPM package for converting markdown or html to AsciiDoc I can add support for this quickly.
